recipe/meta.yaml

```diff
@@ -8,7 +8,7 @@ package:
 
 source:
   url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/sf-hamilton-{{ version }}.tar.gz
-  sha256: 91f71fa81e37efb156ae8901c4b7fc073f794de9ebe7ff8a59db5943de52c0eb
+  sha256: 9dda6ed13c829cedda02eda0713f17995ea23a8511f7a32df61d9744c249ee60
 
 
 build:
```

Comment:
My comments:
1. The `license` name is `BSD-3-Clause-Clear`, see:
- https://github.com/DAGWorks-Inc/hamilton/blob/sf-hamilton-1.35.1/LICENSE
- https://spdx.org/licenses/
2. Add missing flags to `script`: `--no-deps --no-build-isolation`
3. Remove `noarch: python` as we build arch-specific packages
4. Use a plain `python` in `host` and `run`, see also Daniel's comment about skipping python versions
5. Add missing `setuptools` and `wheel` to `host`
6. Missing `license_family`, `description`, `dev_url` (see the linter message)